modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/commerce_order/general.jsp

```diff
@@ -411,9 +416,9 @@ CommerceOrder commerceOrder = commerceOrderEditDisplayContext.getCommerceOrder()
 		/>
 
 		<commerce-ui:panel
-			actionLabel='<%= LanguageUtil.get(request, "edit") %>'
+			actionLabel='<%= commerceOrderEditDisplayContext.hasManageCommerceOrdersPermission() ? LanguageUtil.get(request, "edit") : null %>'
 			actionTargetId="order-summary-modal"
-			actionUrl="<%= editOrderSummaryURL %>"
+			actionUrl="<%= commerceOrderEditDisplayContext.hasManageCommerceOrdersPermission() ? editOrderSummaryURL : null %>"
 			title='<%= LanguageUtil.get(request, "order-summary") %>'
 		>
```

Comment:
Is the update check for his also done using MANAGE_COMMERCE_ORDERS in the MVCActionCommand?

Reply:
@brianikim This one uses this permission as well as MANAGE_COMMERCE_ORDER_PRICE, we include that one as well as a prerequisite for the button to appear right?

Reply:
Only use MANAGE_COMMERCE_ORDER_PRICE

Reply:
@brianikim can you please explain to me why we're only requiring MANAGE_COMMERCE_ORDER_PRICE? Doesn't it make sense to require both of them if both are needed for the user to update it?

Reply:
The most important check is MANAGE_COMMERCE_ORDER_PRICE. Just need to check that one for actually showing the menu.

Reply:
You need to remove this part of the commit completely. and make sure the first change you make is from
https://github.com/brianikim/liferay-portal/pull/107/commits/b9a911ce7a385988395fe36969d2af88a96f35c6